### ISSUE TITLE
Instability..

### ISSUE THREAD

**Issue by caiogrigoleto** [salman0ansari/whatsapp-api-nodejs#724]
It is very unstable, devices keep losing connection..

If the device is inactive for a long time, can it disconnect from the API?

**Comment by caiogrigoleto** [comment]
I noticed that when the cell phone is inactive for a long time it is automatically disconnected from the API, in some cases it keeps switching between connected and disconnected very quickly, I have already looked at the log and it does not show any error or information regarding this disconnection. @joaroque

**Comment by salman0ansari** [comment]
> I noticed that when the cell phone is inactive for a long time it is automatically disconnected from the API, in some cases it keeps switching between connected and disconnected very quickly, I have already looked at the log and it does not show any error or information regarding this disconnection. @joaroque

You have to open whatsapp app on your phone once in 14 days if you don't you device will be logged out.

**Comment by exallhost** [comment]
I'm facing the same problem, the sessions drop in less than approximately 4 hours.

This started to occur when I migrated to Docker. I'll do a test removing it from Docker and return here.

**Comment by caiogrigoleto** [comment]
> I'm facing the same problem, the sessions drop in less than approximately 4 hours.
> 
> This started to occur when I migrated to Docker. I'll do a test removing it from Docker and return here.

How did you manage to solve the problems?

**Comment by solonalbuquerque** [comment]
We are also having issues with sessions definitely crashing from time to time, and also with sessions leaving and coming back.
We use the last code.

**Comment by exallhost** [comment]
> > I'm facing the same problem, the sessions drop in less than approximately 4 hours.
> > This started to occur when I migrated to Docker. I'll do a test removing it from Docker and return here.
> 
> How did you manage to solve the problems?

Yes, after removing Docker I had an improvement of 50%, before 10 accounts dropped today only 5 are dropping.

**Comment by caiogrigoleto** [comment]
> > > I'm facing the same problem, the sessions drop in less than approximately 4 hours.
> > > This started to occur when I migrated to Docker. I'll do a test removing it from Docker and return here.
> > 
> > 
> > How did you manage to solve the problems?
> 
> Yes, after removing Docker I had an improvement of 50%, before 10 accounts dropped today only 5 are dropping.

I'm going to try to do this, run outside of docker ..
